agent/consul/acl_endpoint.go

```diff
@@ -2128,6 +2125,16 @@ func (a *ACL) AuthMethodSet(args *structs.ACLAuthMethodSetRequest, reply *struct
 		}
 	}
 
+	switch method.TokenType {
+	case "local", "":
+	case "global":
+		if !a.srv.InACLDatacenter() {
```

Comment:
Is this only a restriction because the secondary servers would require a token to use in the RPC to the primary?

Reply:
That and having to sleep-loop until replication brought your newly-created token back to the secondary.